### DISCUSSION TITLE
[FR] 关于手动键入时间节点进行视频跳转播放的优化

### DISCUSSION THREAD

**Discussion by XMB-7** [the1812/Bilibili-Evolved#4339] [功能建议 / Ideas]
![Image_20230724_093837](https://github.com/the1812/Bilibili-Evolved/assets/65625632/dea3d34f-8c72-4016-b4d3-cb2674e9c88a)

比如我想跳转到01:30, 目前键入流程是很麻烦的, 首先需要点击播放时长区域, 进入编辑模式, 然后要么ctrl+a全选全删再手动键入01:30回车, 要么移动光标逐一修改完相关位的数字再回车. 这两种方式都比较繁琐.
希望能达成这样一种效果, 比如新增一个插件叫快捷跳转, 当开启后, 点击播放时长区域进入编辑模式后, 直接键入0130四个数字, 就会自动替换掉00:00. (或许可以额外新增一个小弹窗让用户在其内输入, 然后整体进行字符串替换)
最好能连开头的点击播放时长区域进入编辑模式这步都简化了, 比如设置一个快捷键alt+j, 键入alt+j以及0130以及回车, 直接进行跳转播放.

**Comment by Geeyun-JY3** [comment]
破站点击播放时长区域后不直接全选确实麻烦了点。
现在直接输入多位纯数字破站会直接识别成秒数，如输入「0130」会跳转至视频 02:10 处，这功能要做的话在这一块是否还得给个选项？
还有就是别忘了时长大于一个小时的视频。
